### DISCUSSION TITLE
Geniux Music

### DISCUSSION THREAD

**Discussion by geniux-dev** [E-Kuerschner/useAudioPlayer#114] [Show and tell]
Happy to announce of what I have built using this plugin, Lets give @E-Kuerschner a shout!!🔥🔥🔥

Platform: [Geniux Music](https://music.geniux.co.za)

![music geniux co za_playing_4956797468(iPhone 6_7_8 Plus)](https://github.com/E-Kuerschner/useAudioPlayer/assets/47742379/9dbe1c66-eb82-4e82-8d7f-77f38cc45102)

**Comment by E-Kuerschner** [comment]
@geniux-dev this is badass!! I have still yet to use the lib at scale in an application this feature-full. Please keep me up to date with any challenges around using the package and any feedback that could help improve it for everyone!
